content/index.html

```diff
@@ -156,15 +156,16 @@ <h2 class="h1">Publications &amp; Talks</h2>
     </article>
 
     <article class="center community">
-        <h2 class="h1">Community</h2>
+        <h2 class="h1" id="community">Community</h2>
         <div class="wrap">
             <p>libp2p is an open source project from the IPFS community. There are many ways you can contribute by
                 getting involved: you can help write the specification, code implementations and tools using libp2p,
                 craft examples and tutorials, and join
                 our regular product calls to discuss libp2p with us.</p>
```

Comment:
```suggestion
            <p>libp2p is an open source project from the IPFS community. There are many ways you can contribute by
                getting involved: you can help write the specification, code implementations and tools using libp2p, and
                craft examples and tutorials.</p>
```

While we might reintroduce them in the future, we don't do them today.